gslib/commands/mb.py

```diff
@@ -39,8 +39,9 @@
 from gslib.utils.text_util import NormalizeStorageClass
 
 _SYNOPSIS = """
-  gsutil mb [-b (on|off)] [-c <class>] [-l <location>] [-p <project>]
+  gsutil mb [-b (on|off)] [-c <class>] [-l <location>] [-p <proj_id>]
```

Comment:
Is this accidentally undoing an earlier docs change?

Reply:
Good catch!